lib/diagnostic.rb

```diff
@@ -18,7 +33,10 @@
 
 ##
 # your answers here
-Answer.dave = nil
+Answer.dave = dave
+
+dave = Person.new("Dave", 32, "Ohio")
+dave.location = "Somerville"
```

Comment:
Wrong order on this. In the first line `dave` hasn't been defined yet.

Otherwise correct.